packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -79,7 +79,7 @@ export class Span implements types.Span, ReadableSpan {
 
   addEvent(name: string, attributes?: types.Attributes): this {
```

Comment:
Could we also make an optional `time` parameter for the event?

That will be helpful for bringing over the OpenCensus Web code since it generates timed events based on browser performance data (this is similar to my desire to make the start span take a start time and end span take an end time, etc.)

Reply:
This is not defined in the specs: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#add-events. AFAIK SDK is responsible for adding timestamp internally.

Reply:
Hmm, OK. Should I raise an issue with the specs for this? What's the process for that?

I do need the feature for web tracing.

Reply:
so the event timestamp is gonna be different from current timestamp (which we are adding internally) in web tracing? 

> What's the process for that?

Create new issue [here](https://github.com/open-telemetry/opentelemetry-specification/issues) and start the discussion.

Reply:
Thanks for the pointer! I've created https://github.com/open-telemetry/opentelemetry-specification/issues/213 to discuss and shared more about the web tracing use case there (basically generating timed events from Performance API data as measured by the browser runtime rather than in the JS code)

Reply:
Oh huh okay I think that part of the spandata removal is going to have to  include optional timestamps for events, because how else would you do out of band reporting without the timestamp? I'll comment on that issue @draffensperger

Reply:
Thanks for the issue @draffensperger. If all goes well, I will open new PR to merge `TimedEvent` and `Event` interface and add optional timestamp param to `addEvent` method. To unblock #192 work I'd like to merge this one.